### PR TITLE
Create 'istio_ca' docker image

### PR DESCRIPTION
**Pull request by lookuptable**

### CODE REVIEW

**Comment by lookuptable** [comment]
@wattli FYI

File: cmd/istio_ca/main.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		glog.Info("Use self-signed certificate as the CA certificate")
+
+		return certmanager.NewSelfSignedIstioCA()
+	}
+
+	c, k := certmanager.LoadSignerCredsFromFiles(*caCertFile, *caKeyFile)
+	return certmanager.NewIstioCA(c, k)
+}
+
+func generateConfig() *rest.Config {
```
Comment (wattli) [inline cmd/istio_ca/main.go]: Would this generate the proxy config? or just the ca config

Comment (lookuptable) [inline cmd/istio_ca/main.go]: This is the config for talking with the apiserver

Comment (wattli) [inline cmd/istio_ca/main.go]: i see.

**Pull request merged**